platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/NativeMapView.java

```diff
@@ -61,11 +61,8 @@
   // Listener invoked to return a bitmap of the map
   private MapboxMap.SnapshotReadyCallback snapshotReadyCallback;
 
-  //
-  // Static methods
-  //
-
-  static {
+  @Override
+  public void loadNativeLibrary() {
```

Comment:
Don't think this is a valid change, loading the native library is a static operation. cc @ivovandongen